packages/x-date-pickers/src/DateCalendar/DateCalendar.tsx

```diff
@@ -486,7 +486,7 @@ DateCalendar.propTypes = {
   /**
    * Callback fired on focused view change.
    * @template TView
-   * @param {TView} view The new view to focus or not.
+   * @param {TView | null} view The new view to focus or not.
    * @param {boolean} hasFocus `true` if the view should be focused.
```

Comment:
I did not find in which case the view should be null. From what I remember, the `view` is here to say which element is getting/losing the focus, and `hasFocus` indicates if it's getting or losing the focus.

Reply:
Thanks, good observation! 🙌 
It may have been needed initially based on previous implementations, but as it is right now—this change can be reverte. 👌